source/tanya/math/random.d

```diff
@@ -160,6 +160,181 @@ version (linux)
         }
     }
 }
+else version (CRuntime_Bionic) version = secure_arc4random;
```

Comment:
I would prefer to define these versions before the conditional code and then just write
```d
version (CRuntime_Bionic)
{
    version = secure_arc4random;
}
else version (OSX)
{
    version = secure_arc4random;
}
else ...

version (linux)
{
    class PlatformEntropySource
    ...
}
else version (secure_arc4random)
{
    class PlatformEntropySource
    ...
}
```